docs/reference/predicates.md

```diff
@@ -363,6 +363,53 @@ JWTPayloadAllKVRegexp("iss", "^https://")
 JWTPayloadAnyKVRegexp("iss", "^https://")
 ```
 
+### HeaderSHA256
+
+Matches if SHA-256 hash of the header value (secret) equals to any of the configured hash values.
+Several hash values could be used to match multiple secrets e.g. during secret rotation.
```

Comment:
The same could be achieved using two routes with two different `HeaderSHA256` but it is more convenient to use single predicate with multiple secrets to avoid route duplication, e.g. in `zalando.org/skipper-predicate` ingress annotation.